resources/modernizer_exclusions.txt

```diff
@@ -20,3 +20,4 @@ com/google/common/primitives/Longs.compare:(JJ)I
 com/google/inject/Inject
 com/google/inject/Provider
 java/lang/StringBuffer."<init>":()V
+java/util/Hashtable."<init>":(I)V
```

Comment:
> The modernizer plugin forbids the use of Hashtable in favor of HashMap, but HashMap does not extend Dictionary

Ah, like so. Would it be possible/make sense to apply this exclusion to a more limited scope, or do we need to create `Dictionary` objects also in jclouds core for this?

Thanks for explaining, @nacx!